en/admin/admin_utils.rst

```diff
@@ -2326,6 +2339,30 @@ The following shows [options] available with the **cubrid statdump** utility.
     | Num_lfcq_shared_lists                    | Snapshot       | The current number of non-zero candidate shared LRU lists             |
     +------------------------------------------+----------------+-----------------------------------------------------------------------+
 
+    **Double write buffer**
+
+    +------------------------------------------+----------------+-----------------------------------------------------------------------+
+    | **Stat name**                            | **Stat type**  |  **Description**                                                      |
+    +------------------------------------------+----------------+-----------------------------------------------------------------------+
+    | ..Time_DWB_flush_block_time              | Counter/timer  | The number and duration of DWB block flushes                          |
+    +------------------------------------------+----------------+-----------------------------------------------------------------------+
+    | ..Time_DWB_flush_block_helper_time       | Counter/timer  | ??                                                                    |
```

Comment:
Maybe volume prefix may be added. DWB_flush_block_volume_time for flush block daemon, Time_DWB_flush_block_volume_helper_time for flush block helper daemon.